### PR TITLE
Add dev.geopjr.Tuba

### PR DESCRIPTION
**Pull request by GeopJr**
### Please confirm your submission meets all the criteria

- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub.
- [X] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by GeopJr** [comment]
bot, build dev.geopjr.Tuba

File: dev.geopjr.Tuba.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+app-id: dev.geopjr.Tuba
+runtime: org.gnome.Platform
+runtime-version: "44"
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.vala
```
Comment (hfiguiere) [inline dev.geopjr.Tuba.yaml:6]: vala is in the gnome sdk you don't need that. it's not meant for that.

File: dev.geopjr.Tuba.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  - /man
+  - /share/doc
+  - /share/gtk-doc
+  - /share/man
+  - /share/pkgconfig
+  - /share/vala
+  - '*.la'
+  - '*.a'
+modules:
+  - name: libsoup-2.4
```
Comment (hfiguiere) [inline dev.geopjr.Tuba.yaml:26]: you should really port to libsoup 3.

File: dev.geopjr.Tuba.yaml (hunk 1 above)
Comment (GeopJr) [reply dev.geopjr.Tuba.yaml]: Fixed, thanks!

File: dev.geopjr.Tuba.yaml (hunk 2 above)
Comment (GeopJr) [reply dev.geopjr.Tuba.yaml:24]: Tracked https://github.com/GeopJr/Tuba/issues/12

**Comment by TheEvilSkeleton** [comment]
bot, build dev.geopjr.Tuba

**Comment by barthalion** [comment]
bot, build dev.geopjr.Tuba

**Pull request closed without merging**